### PR TITLE
Feature/134431 create metering point data

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Create cosmos db container for metering point and python code that loads metering point from that container

### CODE REVIEW

**Review by kristianschneider — APPROVED** [review]

**Review by ZavezX — APPROVED** [review]

File: source/databricks/aggregation-jobs/aggregation_trigger.py [hunk 1]
```diff
@@ -13,15 +13,17 @@
...
+sys.path.append(r'/workspaces/geh-aggregations/source/databricks')
 
 import json
 import configargparse
 from datetime import datetime
 from geh_stream.aggregation_utils.aggregators import \
     initialize_spark, \
+    load_meteringpoints, \
     load_timeseries_dataframe, \
+    combineDataframes, \
```
Comment (jonasdmoeller) [inline source/databricks/aggregation-jobs/aggregation_trigger.py:26]: use under_scores

File: source/databricks/aggregation-jobs/aggregation_trigger.py [hunk 2]
```diff
@@ -79,7 +84,14 @@
     print("Unknown args: {0}".format(args))
 
 spark = initialize_spark(args)
+
+# meteringpoints = load_meteringpoints(args, spark)
+# meteringpoints.show()
+# meteringpoints.printSchema()
```
Comment (mknic) [inline source/databricks/aggregation-jobs/aggregation_trigger.py:90]: Should we remove these lines?

File: source/databricks/geh_stream/aggregation_utils/aggregators/aggregation_initializer.py [hunk 3]
```diff
@@ -32,6 +32,17 @@ def initialize_spark(args):
         .getOrCreate()
 
 
+def load_meteringpoints(args, spark):
+    readConfigMeteringpoint = {
```
Comment (jonasdmoeller) [inline source/databricks/geh_stream/aggregation_utils/aggregators/aggregation_initializer.py:36]: use under_scores

File: source/databricks/aggregation-jobs/aggregation_trigger.py [hunk 4]
```diff
@@ -79,7 +84,14 @@
...
+combineDataframes = combineDataframes(args, areas, spark)
+combineDataframes.show()
 filtered = load_timeseries_dataframe(args, areas, spark)
+# filtered.show()
```
Comment (mknic) [inline source/databricks/aggregation-jobs/aggregation_trigger.py:94]: Should we remove this line?

**Review by mknic — CHANGES_REQUESTED** [review]
Looks good besides some comments we need to remove.

File: source/databricks/aggregation-jobs/aggregation_trigger.py (hunk 2 above)
Comment (johevemi) [reply source/databricks/aggregation-jobs/aggregation_trigger.py]: comments has been removed

File: source/databricks/aggregation-jobs/aggregation_trigger.py (hunk 4 above)
Comment (johevemi) [reply source/databricks/aggregation-jobs/aggregation_trigger.py]: comment has been removed

**Review by mknic — APPROVED** [review]

**Pull request merged**